sentry-contrib-native-sys/build.rs

```diff
@@ -83,12 +105,35 @@ fn build(out_dir: &Path, source: &Path, install: &Path) -> Result<()> {
         .context("failed to parse path")?
         .to_owned();
 
-    if !Command::new("cmake")
-        .current_dir(source)
-        .args(&["-B", &build, "-D", "BUILD_SHARED_LIBS=OFF"])
-        .status()?
-        .success()
+    let mut cfg_cmd = Command::new("cmake");
+    // Build static libraries
+    cfg_cmd.args(&[
+        "-B",
+        &build,
+        "-D",
+        "BUILD_SHARED_LIBS=OFF",
+        "-D",
+        "SENTRY_BUILD_TESTS=OFF",
+        "-D",
+        "SENTRY_BUILD_EXAMPLES=OFF",
```

Comment:
This are off by default, no?
https://github.com/getsentry/sentry-native/blob/master/CMakeLists.txt#L35-L36
```
option(SENTRY_BUILD_TESTS "Build sentry-native tests" "${SENTRY_MAIN_PROJECT}")
option(SENTRY_BUILD_EXAMPLES "Build sentry-native example(s)" "${SENTRY_MAIN_PROJECT}")
```
https://cmake.org/cmake/help/latest/command/option.html

> If no initial `<value>` is provided, `OFF` is used.

Reply:
Maybe it changed, but when I was watching the verbose build output for the first few times I was building I definitely saw tests and examples being compiled and it stopped after I added those defines.

Reply:
I looked into this, `"${SENTRY_MAIN_PROJECT}"` is the default value and is being set to `ON`, not gonna  try to get into CMake again, but yeah, this is good.